docs/_data/meetups/usa-portland.yaml

```
@@ -0,0 +1,6 @@
+region: North America
+country: USA
+state: Oregon
+city: Portland
+notes: |
+  Organised by [@WriteTheDocsPDX](https://twitter.com/WriteTheDocsPDX)
```

This seems broken because there is no meetup website. We should raise an error in the extension here, I think?

---

:+1:

---

Probably we should require Region / Country / City / website

---

So I don't have access to sphinxerror in conf.py? in L137

```py
if not 'website' in meetup:
            raise SphinxError('Meetup needs a website')
```

Results in

```
Exception occurred:
  File "conf.py", line 138, in load_meetups_by_region
    raise SphinxError('Meetup needs a website')
NameError: name 'SphinxError' is not defined
```

---

Where are you expecting SphinxError to come from?

---

I lazily googled errors in sphinx, and expected to have access to it within conf.py